### PR TITLE
Adds a Gadpathurian kepi.

### PR DESCRIPTION
**Pull request by TheStryker**
Adds a Gadpathurian patrol cap in the style of a kepi under the gadpathurian headwear section in the loadout tab.
As requested by Schwann.
image

### CODE REVIEW

**Comment by TheStryker** [comment]
!review

**Review by KingOfThePing — APPROVED** [review]

File: code/modules/clothing/factions/gadpathur.dm [hunk 1]
```diff
@@ -89,6 +89,24 @@
 	icon_state = "gadpathur_medic_beret"
 	item_state = "gadpathur_medic_beret"
 
+/obj/item/clothing/head/ushanka/gadpathur
+	name = "gadpathurian patrol cap"
```
Comment (listerla) [inline code/modules/clothing/factions/gadpathur.dm:93]:
```suggestion
	name = "\improper Gadpathurian patrol cap"
```

**Review by Arrow768 — APPROVED** [review]

File: code/modules/clothing/factions/gadpathur.dm [hunk 2]
```diff
@@ -89,6 +89,24 @@
...
+	item_state = "gadpathur_kepi"
+	icon = 'icons/clothing/head/gadpathur_kepi.dmi'
+	contained_sprite = TRUE
+
+/obj/item/clothing/head/ushanka/gadpathur/attack_self(mob/user as mob)
+	src.earsup = !src.earsup
+	if(src.earsup)
+		icon_state = "[initial(icon_state)]_up"
+		item_state = "[initial(icon_state)]_up"
+		to_chat(user, "You raise the sun shade on the cap.")
```
Comment (NonQueueingMatt) [inline code/modules/clothing/factions/gadpathur.dm:105]:
```suggestion
		to_chat(user, SPAN_NOTICE("You raise the sun shade on the cap."))
```

File: code/modules/clothing/factions/gadpathur.dm [hunk 3]
```diff
@@ -89,6 +89,24 @@
...
+	else
+		src.icon_state = initial(icon_state)
+		to_chat(user, "You lower the sun shade on the cap.")
```
Comment (NonQueueingMatt) [inline code/modules/clothing/factions/gadpathur.dm:108]:
```suggestion
		to_chat(user, SPAN_NOTICE("You lower the sun shade on the cap."))
```

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**